internal/configtxgen/encoder/encoder_test.go

```diff
@@ -46,7 +46,115 @@ func CreateStandardPolicies() map[string]*genesisconfig.Policy {
 	}
 }
 
+func CreateStandardOrdererPolicies() map[string]*genesisconfig.Policy {
+	return map[string]*genesisconfig.Policy{
+		"Admins": {
+			Type: "ImplicitMeta",
+			Rule: "ANY Admins",
+		},
+		"Readers": {
+			Type: "ImplicitMeta",
+			Rule: "ANY Readers",
+		},
+		"Writers": {
+			Type: "ImplicitMeta",
+			Rule: "ANY Writers",
+		},
+		"BlockValidation": {
+			Type: "ImplicitMeta",
+			Rule: "ANY Admins",
+		},
+	}
+}
+
```

Comment:
Is there a reason we didn't call CreateStandardPolicies and then append a `BlockValidation` policy?

Reply:
oversight, will update 👍